packages/matter-node.js/src/matter/certificate/RootCertificateManager.ts

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
Is this existing functionality or new functionality unrelated to persistence?  It's best to keep PRs focused on one thing.

Reply:
In fact yes it is unrelated. I switched it out and then added persistence ... Would be cool to include here still because just moved code without any other change then persistence.